Gemfile

```diff
@@ -45,6 +45,9 @@ gem 'jbuilder', '~> 2.5'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+# Enables the creation of both simple and advanced search forms
+gem 'ransack'
```

Comment:
Please include version numbers for new gems (similar to the other gems in the Gemfile).